src/dmd/backend/cgcod.d

```diff
@@ -579,6 +583,10 @@ tryagain:
         }
         sfunc.Ssize = Offset(sfunc.Sseg) - funcoffset;    // size of function
 
+        //xyzzy disasm
```

Comment:
What is "xyzzy"?

Reply:
a piece of debug code I forgot to remove